docs/content/documentation/use.md

```diff
@@ -116,6 +116,8 @@ This value is an optional argument to the constructor of the sslfactory class pr
 * **`sslmode (`*String*`)`** *Default `prefer`*\
 possible values include `disable` , `allow` , `prefer` , `require` , `verify-ca` and `verify-full` . `require` , `allow` and `prefer` all default to a non-validating SSL factory and do not check the validity of the certificate or the host name. `verify-ca` validates the certificate, but does not verify the hostname. `verify-full` will validate that the certificate is correct and verify the host connected to has the same hostname as the certificate. Default is `prefer` Setting these will necessitate storing the server certificate on the client machine see [Configuring the client](/documentation/ssl/#configuring-the-client) for details.
 
+* **`sslNegotiation (`*String*`)`** *Default `postgres` *\
+possible values include `postgres` and `direct`. `postgres` is the default and traditional SSL negotiation is performed where GSS is requested, then SSL is requested. If set to `direct` then SSL is assumed and an SSL packed is sent without requesting if the server supports it. This avoids one round trip to the server and is only available for server version 17 or higher.
```

Comment:
I wonder if this setting should interconnect with `assumeMinServerVersion`.

For instance, if user selects `assumeMinServerVersion=17`, we could assume `sslNegotiation=postgres`.
At the same time, if user selects `sslNegotiation=postgres`, we could select `assumeMinServerVersion=17` to make startup packets faster.

WDYT?

Reply:
how does serverVersion 17 make startup packets faster?
Not sure I like presuming `sslNegotiation=direct` if version 17 as if for some reason it doesn't connect it will fail